generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -90,7 +93,32 @@ export class <%= entityAngularJSName %>DialogComponent implements OnInit {
         <%- queries[idx] %>
         <%_ } _%>
     }
+    <%_ if (fieldsContainBlob) { _%>
+    byteSize(field) {
+        return this.dataUtils.byteSize(field);
+    }
 
+    openFile(contentType, field) {
+        return this.dataUtils.openFile(contentType, field);
+    }
+    <%_ } _%>
+    <%_ for (idx in fields) {
+        if ((fields[idx].fieldType === 'byte[]' || fields[idx].fieldType === 'ByteBuffer') && fields[idx].fieldTypeBlobContent !== 'text') { _%>
+    set<%= fields[idx].fieldNameCapitalized %>($event, <%= entityInstance %>) {
+        if ($event.target.files && $event.target.files[0]) {
```

Comment:
for multiple fields this will be repeated can we move the common boilerplate code into a private helper method and pass the filed specific stuff below as a callback?

I would even consider writing a generic function like below as I hate to see duplicated code and pass the filedName and a boolean if its image as below
```
setFileData($event, <%= entityInstance %>, field, isImage) {
  if ($event.target.files && $event.target.files[0]) {
    let $file = $event.target.files[0];
    if (isImage && !/^image\//.test($file.type)) {
      return;
    }
    this.dataUtils.toBase64($file, (base64Data) => {
      <%= entityInstance %>[field] = base64Data;
      <%= entityInstance %>[`${field}ContentType`] = $file.type;
    }
  }
}
```